firefox-focus-monitor.py

```diff
@@ -0,0 +1,94 @@
+import argparse
```

Comment:
> I use PEP-636 style Structural Pattern Matching which means it relies on having at least Python 3.10.

Let's add some comments about the minimum requirements for the Python3 version.

If there's a lot of python scripts in this repository later. I will add some minimum requirements around the whole project.

Reply:
just thought that a lot of the value from "contrib" style repos is that it's easy to grab odd bits of the code out for inspiration for your own use case.  having something too polished and packaged up seems to go against that...

Reply:
I agree, but I also don't want this repository to have too many broken things.

I won't impose strict rules, but let's ensure that people know how to use it and establish minimum requirements for this script.

The minimum requirement for this repository is just an idea. I guess it would be better not to have it.

So, I suggest you add some comments regarding the Python version requirements, like:
` python --version >= Python 3.10.0`
Also, please copy the comments from your commit and paste them at the top of your script.

Reply:
Would be easy to set up some CI actions for automatic linting of code.  https://github.com/astral-sh/ruff seems to be the fashionable option for Python at the moment and I can confirm it works well with Github actions (and is easy to run locally as well)

Reply:
Oh thanks, that's what I've been looking for.